server/services/participants.js

```diff
@@ -71,6 +71,32 @@ const setParticipantStatus = async (
   return { status };
 });
 
+const getParticipantByID = async (participantInfo) => dbClient.db.withTransaction(async (tx) => {
+  const participant = await tx[collections.PARTICIPANTS].findDoc({
+    id: participantInfo.id,
+  });
+  return participant;
+});
+
+const updateParticipant = async (participantInfo) => dbClient.db.withTransaction(async (tx) => {
```

Comment:
This totally seems like it will accomplish what we want but I'm concerned about the safety of taking a user-submitted change object and modifying the record. I wonder if we should filter out props that we definitely don't want changed (`id`, `maximusId`, etc.)?

Reply:
Just saw that this is actually dependent on the validation which only lets certain properties through. Therefore, at the time of writing, only first name, last name, phone number, and email address are editable.